articles/function-declarations-and-calls.html

```diff
@@ -141,7 +141,7 @@ <h3>Function Declarations</h3>
 
 Here are more function declaration examples.
 <pre class="line-numbers"><code class="language-go">func CompareLower4bits(m, n uint32) (r bool) {
-	// The following two lines <=> return m&0xFF > n&0xff
+	// The following two lines <=> return m&0xF > n&0xf
 	r = m&0xF > n&0xf

```

Comment:
It is my fault. It would be better to make the two `0xF` consistent.
```
r = m&0xF > n&0xF
```

Reply:
Got it